ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/README.md

```diff
@@ -0,0 +1,26 @@
+# Clip Geometry
+
+This sample demonstrates how to clip a geometry with an envelope using the GeometryEngine.
+
+![](screenshot.png)
+
+Click or tap the "Clip" button to clip the blue graphic with the red envelopes.
+
+## How it works
+1. Create a `graphicsOverlay` containing an envelope.
+2. Create another that contains three other envelopes: one that intersects a part of the first graphic, one that is entirely within it, and another completely outside of it.
+3. Create one more `graphicsOverlay` that contains no graphics.
```

Comment:
Remove `that contains no graphics`. That's the default behavior when creating a GraphicsOverlay.